src/FockSpace/ONV.cpp

```diff
@@ -32,28 +32,32 @@ namespace GQCP {
  *  @param N                        the number of electrons
  *  @param unsigned_representation  the representation for the ONV as an unsigned integer
  */
-ONV::ONV(size_t K, size_t N, size_t unsigned_representation):
-    K (K),
-    N (N),
-    unsigned_representation (unsigned_representation)
+ONV::ONV(size_t K, size_t N, size_t unsigned_representation) :
+    ONV(std::make_tuple(K, N))
 {
-    occupation_indices = VectorXs::Zero(N);
+    this->unsigned_representation = unsigned_representation;
     this->updateOccupationIndices();  // throws error if the representation and N are not compatible
 }
 
 /**
  *  @param K                        the number of orbitals
  *  @param unsigned_representation  the representation for the ONV as an unsigned integer
  */
-ONV::ONV(size_t K, size_t representation):
-    ONV(K, __builtin_popcountl(representation), representation)
+ONV::ONV(size_t K, size_t representation) :
+        ONV(K, __builtin_popcountl(representation), representation)
```

Comment:
remove excessive indentation
```suggestion
    ONV(K, __builtin_popcountl(representation), representation)
```